lib/ember-cli/manifest.rb

```diff
@@ -0,0 +1,55 @@
+module EmberCli
+  class Manifest
+    class Null
+      def fixup!
+      end
+
+      def files
+        {}
+      end
+
+      def assets
+        {}
+      end
+    end
+
+    def initialize(manifest:, app_name:)
+      @app_name = app_name
+      @manifest = manifest
+      @assets = {}
+      @files = {}
+    end
+
+    def fixup!
+      fixup_files!
+      fixup_assets!
+
+      self
+    end
+
+    def files
+      @files
+    end
+
+    def assets
+      @assets
+    end
+
+    private
+
+    def fixup_files!
+      @manifest.files.each do |(file, info)|
+        fixed_info = info.merge(
+          "logical_path" => "#{@app_name}/#{info["logical_path"]}"
```

Comment:
Prefer single-quoted strings inside interpolations.<br>Put a comma after the last parameter of a multiline method call.